types/sanitize-html/index.d.ts

```diff
@@ -12,64 +12,56 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.2
 
-///<reference types="htmlparser2"/>
-
-import { Options } from "htmlparser2";
+import { ParserOptions, DomHandlerOptions } from 'htmlparser2';
```

Comment:
`ParserOptions` and `DomHandlerOptions` do not appear to be used in the code below, whereas the deleted `Options` is.

Reply:
Fixed! Thanks for the catch, I had changed it in the local version I was testing with but forgot to copy that line over. :( With whitelist not updated yet and already getting TSC errors due to htmlparser2 not being node_modules my editor isn't catching my ass on this